screenshots/test.py

```diff
@@ -0,0 +1,57 @@
+from seltest import url, waitfor, hide, Base
+from selenium.webdriver.support.ui import Select, WebDriverWait
+import sys
+
+PILEUP_RENDERED_JS = '''
+return document.querySelector('.track.pileup canvas') && document.querySelector('.track.pileup canvas').height > 100
+'''
+
+def _pileup_is_rendered(driver):
+    return driver.execute_script(PILEUP_RENDERED_JS)
+
+
+def wait_for_pileup(driver):
+    driver.implicitly_wait(0)
+    WebDriverWait(driver, 120).until(_pileup_is_rendered, 'pileup timed out')
+    driver.implicitly_wait(120)
+
+
+class Pileup(Base):
```

Comment:
filed https://github.com/hammerlab/seltest/issues/62 as well to make this easier in the future